### ISSUE TITLE
Error: Can't resolve 'is-immutable' 

### ISSUE THREAD

**Issue by xingxing888** [whawker/react-jsx-highcharts#132]
#### Description
I tried to test out the example series inside my react application. The sample model is https://stackblitz.com/github/whawker/react-jsx-highcharts-examples?file=src%2FCombo%2FCombo.js
however, the program doesn't even compile. the error message is:
```
bundle.js:72874 ./~/react-jsx-highcharts/dist/es/components/Series/Series.js
Module not found: Error: Can't resolve 'is-immutable' in '/Users/jiahong.sun/dev/temp/dashboard/node_modules/react-jsx-highcharts/dist/es/components/Series'
 @ ./~/react-jsx-highcharts/dist/es/components/Series/Series.js 15:0-39
 @ ./~/react-jsx-highcharts/dist/es/components/Series/index.js
 @ ./~/react-jsx-highcharts/dist/es/index.js
 @ ./src/containers/UploadsVis.jsx
 @ ./src/containers/ESView.jsx
 @ ./src/routes.js
 @ ./src/containers/Root.js
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://local.climate.com:7776 webpack/hot/dev-server babel-polyfill ./src/index.js
```
what i installed are:
npm install --save react-jsx-highcharts
npm install --save react react-dom prop-types highcharts@^6.0.0
npm install --save highcharts

my package.json looks like:
{
  "name": "dashboard",
  "version": "0.0.1",
  "description": "Debugging dashboard for 3rd party uploads",
  "main": "src/index.js",
  "dependencies": {
    "bootstrap-daterangepicker": "^2.1.25",
    "elasticsearch": "^15.0.0",
    "highcharts": "^6.1.1",
    "isomorphic-fetch": "^2.2.1",
    "moment": "^2.18.1",
    "prop-types": "^15.6.2",
    "react": "^15.6.2",
    "react-bootstrap": "^0.31.2",
    "react-bootstrap-datetimerangepicker": "^2.0.4",
    "react-confirm-alert": "^2.0.5",
    "react-dom": "^15.6.2",
    "react-jsx-highcharts": "^3.1.0",
    "react-redux": "^5.0.4",
    "react-router": "^3.0.5",
    "react-router-dom": "^4.1.1",
    "react-router-redux": "^4.0.8",
    "react-select": "^1.2.1",
    "redux": "^3.6.0",
    "redux-form": "^7.4.2",
    "redux-form-website-template": "0.0.123",
    "redux-logger": "^3.0.6",
    "redux-thunk": "^2.2.0",
    "uuid": "^3.1.0"
  },
  "devDependencies": {
    "@tcc/eslint-config": "^1.1.3",
    "autoprefixer": "^6.4.1",
    "babel-core": "^6.24.1",
    "babel-eslint": "^7.2.2",
    "babel-loader": "^6.4.1",
    "babel-polyfill": "^6.23.0",
    "babel-preset-es2015": "^6.24.1",
    "babel-preset-es2015-loose": "^8.0.0",
    "babel-preset-react": "^6.24.1",
    "babel-preset-stage-0": "^6.24.1",
    "chai": "^3.5.0",
    "css-loader": "^0.28.0",
    "enzyme": "^2.8.2",
    "eslint": "^4.3.0",
    "eslint-plugin-babel": "^5.1.0",
    "eslint-plugin-react": "^7.2.1",
    "eslint-plugin-webdriverio": "^1.0.1",
    "jsdom": "^9.12.0",
    "less": "^2.7.2",
    "less-loader": "^4.0.3",
    "mocha": "^3.2.0",
    "npm": "^4.5.0",
    "path": "^0.12.7",
    "react-addons-test-utils": "^15.5.1",
    "sinon": "^2.1.0",
    "style-loader": "^0.16.1",
    "webpack": "^2.4.1",
    "webpack-dev-server": "^2.4.4"
  },

}

#### Versions
* React JSX Highcharts version: 
* Highcharts version: 
* React version:

**Comment by whawker** [comment]
Hi @xingxing888, dang I accidentally published is-immutable@1.0.2 without building the code first, silly me!

I've just published 1.0.3 which should fix the issue. I think `npm update is-immutable` might sort it?

P.s. As an aside it's worth noting that support for React 15 was dropped in React JSX Highcharts V3, and it looks like you're using React ^15.6.2 in your `package.json`

**Comment by whawker** [comment]
If `npm update is-immutable` doesn't update the package, try `npm add is-immutable && npm remove is-immutable`
